### ISSUE TITLE
User guide: Thesis student / tudatpy from source

### ISSUE THREAD

**Issue by jo11he** [tudat-team/tudat-space#48]
User guide for people who want to use tudatpy from source, extending functionalities through custom python code or exposure of (new/modified/unexposed) c++ source. 

One big question @FilippoOggionni, @DominicDirkx: where will we place this page?

**Comment by FilippoOggionni** [comment]
I think it would fit better into the developer documentation... But we can surely include a note in tudat-space with a link. What do you guys think? 🙂

**Comment by jo11he** [comment]
Yes, you could be right. Considering how much material from the Dev Docs I will have to refer to, it suggests that the hole page belongs there. Will have to talk to @ggarrett13 where he sees fit for it within the Dev Docs structure...
